listener/metrics.go

```diff
@@ -23,10 +23,16 @@ var (
 		Subsystem: "listener",
 		Name:      "kafka_message_received",
 	}, []string{"event", "type"})
+
+	messageHandlingDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: "patchman_engine",
+		Subsystem: "listener",
+		Name:      "kafka_message_handling_duration_seconds",
```

Comment:
Missing buckets, I'd also unify the names between listener and evaluator, something like `handle_duration`

Reply:
We can use default buckets here. This is the convention according to Prometheus docs to add duration_{unit} suffix. I wouldn't add "handle" word to evaluation duration metric name.